App/src/shared/Header.tsx

```diff
@@ -57,6 +59,11 @@ export default function Header() {
 const DesktopNav = () => {
     const linkHoverColor = useColorModeValue("gray.800", "white");
     const popoverContentBgColor = useColorModeValue("white", "gray.800");
+    const navigate = useNavigate();
+    function loggingOut() {
```

Comment:
Log out works nicely, but in the future please make separate issues and PRs for separate tasks, for better organization, and so that we can merge things incrementally. We can't merge this new logout functionality until unrelated admin forms are fixed.

Reply:
Why not exactly? It works doesn't it?

Reply:
Yes, but it has nothing to do with admin forms, it's a different story. We can't merge this working chunk of code until unrelated parts are finished too, which means there's risk of you falling behind the state of the application already in main and the rest of the team falling behind what you're working on separately and making conflicting changes. In general, it's easier to work in bite-size chunks